crypto/ecdh_extra/ecdh_tests.txt

```diff
@@ -2,6 +2,209 @@
 # http://csrc.nist.gov/groups/STM/cavp/documents/components/ecccdhtestvectors.zip
 #
 # P-521 test vectors were fixed to have the right length.
+#
```

Comment:
Are there test vectors we can add from the standards?
I have the same question about the sign and verify tests.

Reply:
Nope, unfortunately there are no test vectors in the standards.